react/src/components/Weather/Forecast/Forecast.js

```diff
@@ -0,0 +1,101 @@
+
+const Forecast = ({ data }) => {
+    const dayInAWeek = new Date().getDay();
+    const forecastDays = WEEK_DAYS.slice(dayInAWeek, WEEK_DAYS.length).concat(WEEK_DAYS.slice(0, dayInAWeek));
+    console.log("data", data)
+    if (!data) return null;
+
+    return (
+        <>
+            <div className={ForecastStyle.title}>Pogoda długoterminowa</div>
+            <Accordion allowZeroExpanded className={ForecastStyle.container}>
+                {data.list.splice(0, 5).map((item, idx) => (
+                    <AccordionItem key={idx}>
+                        <AccordionItemHeading>
+                            <AccordionItemButton className={ForecastStyle.wrapper}>
+                                <div className={ForecastStyle.daily_item}>
+                                    <div className={ForecastStyle.weather}>
+                                        <img src={ICONS[item.weather[0].main]} />
+                                    </div>
+                                    <div className={ForecastStyle.day}>{forecastDays[idx]}</div>
+                                    <div className={ForecastStyle.min_max}>
+                                    <DisplayTemp 
+                                        temp={item.main.temp.toLocaleString(undefined, {
+                                            maximumFractionDigits: 0,
+                                        })}
+                                    />
+                                    °C  
+                                    </div>                              
+                                </div>
+                            </AccordionItemButton>
+                        </AccordionItemHeading>
+                        <AccordionItemPanel className={ForecastStyle.wrapper}>
+                            <div className={ForecastStyle.daily_details_grid}>
+                                <span className={ForecastStyle.daily_details_grid_item}>
+                                    Ciśnienie:{" "}
+                                    <DisplayPressure pressure={item.main.pressure} /> hPa
+                                </span>
+                                <div className={ForecastStyle.daily_details_grid_item}>
+                                    Wilgotność:{" "}
+                                    <DisplayHumidity humidity={item.main.humidity} /> %
+                                </div>
+                                <div className={ForecastStyle.daily_details_grid_item}>
+                                    Zachmurzenie: <DisplayData clouds={item.clouds.all} /> %{" "}
+                                </div>
+                                <div className={ForecastStyle.daily_details_grid_item}>
+                                    Wiatr:{" "}
+                                    <DisplayData
+                                        wind={(item.wind.speed * 3.6).toLocaleString(undefined, {
+                                            maximumFractionDigits: 0,
```

Comment:
Oj. Ciężkie obliczenia jak na templakę. Takie coś już powinno być zaadresowane albo w `DisplayData` albo powinniśmy mieć jakąś funkcję, na górze tego pliku, która nam to przeliczy:

Na przykład
```javascript
formatWindData = function(wind) {
    return (item.wind.speed * 3.6)
                    .toLocaleString(undefined, {maximumFractionDigits: 0})
}

<DisplayData wind={formatWindData(item.wind.speed)}>

Reply:
mój plik, nie moje obliczenia. proszę o dodanie uwagi na branch feature/pogodamenu